screens/Home.tsx

```diff
@@ -1,154 +1,110 @@
-import React, {useEffect, useState} from "react";
-import { GluestackUIProvider, Text, Box } from "@gluestack-ui/themed";
-import { Button, ButtonText, ButtonIcon, AddIcon } from "@gluestack-ui/themed";
-import { useRoute } from "@react-navigation/native";
-import { logout } from "../redux/actions";
-import { useAppDispatch, useAppSelector } from '../redux/hooks';
-import { useNavigation } from "@react-navigation/native";
-import { StackNavigationProp } from "@react-navigation/stack";
+import React, { useEffect, useState } from "react";
 import { SafeAreaView } from "react-native-safe-area-context";
-import { Dimensions, View, StyleSheet } from 'react-native'
+import { Dimensions, View, StyleSheet, ActivityIndicator } from "react-native";
+import { useFonts, PressStart2P_400Regular } from "@expo-google-fonts/press-start-2p";
+import { useAppSelector } from "../redux/hooks";
+import PacmanBackground from "../assets/SVGs/home/PacmanBackground.svg";
+import { getCurrentOrNext } from "../api/getCurrentNextEvent";
 import { RootState } from "../redux/store";
-import EvilIcons from "@expo/vector-icons/EvilIcons";
+import CurrentEventCard from "../Components/CurrentEventCard";
 import Colors from "../constants/Colors";
-import {
-  useFonts,
-  PressStart2P_400Regular,
-} from "@expo-google-fonts/press-start-2p";
-
-type RootStackParamList = {
-  Home: undefined;
-  Login: undefined;
-};
-
-import OngoingEvent from "../assets/SVGs/home/ongoing_event.svg"
-import Background from "../assets/SVGs/home/home_bg.svg"
-import Token from "../assets/SVGs/home/token.svg"
-
-import { getCurrentOrNext } from "../api/getCurrentNextEvent";
-import { getEvents } from "../api/getEvents";
 
+const { width, height } = Dimensions.get("window");
 
-
-const {width, height} = Dimensions.get("window")
-type HomeScreenNavigationProp = StackNavigationProp<RootStackParamList, 'Home'>;
-
-const Home: React.FC = ({}) => {
+const Home: React.FC = () => {
   let [fontsLoaded] = useFonts({
-    PressStart2P_400Regular
+    PressStart2P_400Regular,
   });
   const token = useAppSelector((state: RootState) => state.token);
   const [currNextEvent, setCurrNextEvent] = useState(null);
+  const [loading, setLoading] = useState(true); // To track loading state
 
   useEffect(() => {
-    const fetchCurrNext = async() => {
+    const fetchCurrNext = async () => {
       try {
         const event = await getCurrentOrNext(token);
         setCurrNextEvent(event);
+        console.log("NEXT/CURRENT EVENT", event);
```

Comment:
just remove the console logs and this should be good!

Reply:
done